api/lib/domain/read-models/SchoolingRegistrationForAdmin.js

```diff
@@ -0,0 +1,29 @@
+class SchoolingRegistrationForAdmin {
```

Comment:
Je ne trouve pas de tests associés, du coup je suis allé voir ce qui se faisait du côté des autres read-models.

Sur `api/tests/unit/domain/read-models/CampaignParticipationInfo_test.js` par exemple, le constructeur est testé et on remarque qu'il y a une validation dans l'implem.
`validateEntity(validationSchema, this);`

Je pense qu'il faudrait rajouter cette validation et rajouter un fichier de test pour ce nouveau read-model

Reply:
J'ai rajouté un test et une validation du model.